inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, args: list[str], broker: DispatcherBroker, install_check_serv
         self._dispatcher_broker = broker
         self._install_check_service = install_check_service
         self.update_queue: Queue[Tuple[str, str, Optional[str]]] = Queue(1)
-        self._thread_count = 1
+        self._thread_count = 2
```

Comment:
<h3>$\color{#15F4EE}Miscellaneous$</h3>

Ensure that increasing `_thread_count` to 2 is aligned with the system's concurrency requirements and does not introduce unintended side effects.

Reply:
We'll need to find another way to fix this issue.  We can't increase this or the system could try and do a FOTA and SOTA at the same time.

Reply:
The root cause of the thread increment is because of the apscheduler. The job thread only will be created after a job being added. Hence it increases the active thread count.
The job list is empty while creating the apscheduler. The apscheduler doesn't activate the job thread at the beginning. This causes the active_start_count to not count on the apscheduler thread.
To fix the issue, I created a dummy job to print the message when starting the apscheduler. The message will print out after 1s. This activates the apscheduler to create the job thread, and this thread is counted into active_start_count now.